packages/tests-proxy/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+	name: "velocity:test-proxy",
+	summary: "Dynamically created package to expose test files to mirrors",
+	version: "0.0.4",
+	debugOnly: true
+});
+
+Package.onUse(function (api) {
+	api.use("coffeescript", ["client", "server"]);
+	api.add_files("tests/mocha/client/sampleClientTest.js",["client"]);
+	api.add_files("tests/mocha/server/HelpersTest.js",["server"]);
+	api.add_files("tests/mocha/server/sampleServerTest.js",["server"]);
+});
```

Comment:
nitpicks: Hmm... I thought our linter checked for tabs, but it probably doesn't look in the packages directory. Could you change these two 2 spaces, and add a trailing new line to the file?